src/components/Card.js

```diff
@@ -46,6 +46,18 @@ export const Card = () => {
     }
   };
 
+  const isWEPWithPasswordLengthShorterThat5Characters = () => {
```

Comment:
I'm not sure about this name of function. Could be better?

Reply:
I'm not against the func name or having it factored out, but it's only applicable to `disableHidePassword` so maybe just move it into that function?

```js
const disableHidePassword = () => {
    const isWEPWithPasswordLengthShorterThat5Characters = () => { ... }
    return ...
}
```

Reply:
Perfect! Very good idea. Thanks! It's done